docs/getting-started/mapping-vehicle-data.md

```diff
@@ -38,8 +38,8 @@ important there is a required name `/vehicle_pose`. The list of
 [category values](/docs/protocol-schema/session-protocol.md#stream_metadata) can be seen in the
 specification.
 
-On every frame we must define the current Pose. We use the
-[XVIZBuilder](/docs/api-reference/xviz-builder.md#pose-streamid-) to create this for each frame.
+On every message we must define the current Pose. We use the
```

Comment:
Do we really need the post for every message?

Reply:
I changed this to add 'currently' as we plan to remove this requirement.
https://github.com/uber/xviz/issues/468